<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D3WindowDBManagerClass</class>
 <widget class="QMainWindow" name="D3WindowDBManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>D3WindowDBManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buildWndListButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>103</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>(re)build window list</string>
    </property>
   </widget>
   <widget class="QComboBox" name="windowsComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="flashWindowButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>flash window</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>windows per row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restoreButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>restore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shrinkButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>shrink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tileButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tile</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="windowsPerRowSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
